### PR TITLE
feat: OPX-3.2.0

### PR DESCRIPTION
**Pull request by Tejaswi-Goel**
* Bugfix: fix Coverity high severity errors in the nas-common repo
* Bugfix: traffic is not egressed from DUT after applying Policy Route Map in s5148f Platform
* Update: API for bridge id to vlan ID mapping

Signed-off-by: Tejaswi Goel <Tejaswi_Goel@Dell.com>

### CODE REVIEW

File: inc/opx/nas_com_bridge_utils.h [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline inc/opx/nas_com_bridge_utils.h]: change copyright plz

File: src/nas_com_bridge_utils.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2018 Dell Inc.
```
Comment (rakeshdatta) [inline src/nas_com_bridge_utils.cpp]: change copyright plz

**Review by rakeshdatta — CHANGES_REQUESTED** [review]

**Comment by Tejaswi-Goel** [comment]
Pull-request updated, HEAD is now 46eb01b12ee544b3dcdba5359a64c79953d4a19e

File: src/nas_com_bridge_utils.cpp (hunk 2 above)
Comment (Tejaswi-Goel) [reply src/nas_com_bridge_utils.cpp]: done

**Review by rakeshdatta — APPROVED** [review]

**Pull request merged**